adr/ADR-43.md

```diff
@@ -70,3 +69,7 @@ type StreamConfig struct {
 	LimitsTTL   time.Duration `json:"limits_ttl"`
```

Comment:
`LimitsTTL` has a lower bound of 1 second. Config update fails otherwise.

Reply:
updated